discord/ext/commands/bot.py

```diff
@@ -938,11 +957,11 @@ class be provided, it must be similar enough to :class:`.Context`\'s
 
         invoker = view.get_word()
         ctx.invoked_with = invoker
-        ctx.prefix = invoked_prefix
+        ctx.prefix = invoked_prefix  # type: ignore
```

Comment:
Function would need rework to remove this, as `invoked_prefix` type changes from `List[str] | str` to `str` throughout function.